README.md

```diff
@@ -120,7 +120,7 @@ Note that even if enry's CLI is compatible with linguist's, its main point is th
 Java bindings
 ------------
 
-Generated Java binidings using a C shared library + JNI are located under [`java`](java)
+Generated Java binidings using a C shared library + JNI are located under [`java`](https://github.com/src-d/enry/blob/master/java)
```

Comment:
`docsrv` cannot infer relative references? Shouldn't it be better to add support to infer them instead of forcing every repo to avoid them? (which will be hard)

Reply:
It could be proposed as a feature request, but for this first release docsrv only converts `md` to `html` instead of parsing/replacing links